src/ui/Facet/ValueElementRenderer.ts

```diff
@@ -181,7 +181,7 @@ export class ValueElementRenderer {
   }
 
   private initAndAppendLabel() {
-    this.label = $$('label', { className: 'coveo-facet-value-label' }).el;
+    this.label = $$('div', { className: 'coveo-facet-value-label' }).el;
```

Comment:
How do we make sure that assistive technology will read out the facet value correctly if it cannot be a label ? 

Do we need to add some aria attributes ?

Reply:
Currently, the `tabindex` for those elements only includes the select checkbox and exclude button, which both already have an `aria-label`. This label was redundant.